### PR TITLE
Improve code coverage for SNS rebalance

### PR DESCRIPTION
**Pull request by JugalPatil**
Added steps for rebalance in SNS repair UT to improve code coverage for SNS rebalance
Renamed sns-cm-repair-ut to sns-cm-repreb-ut
For code coverage improvement analysis please refer CORTX-30098.

Signed-off-by: Jugal Patil <jugal.patil@seagate.com>

### CODE REVIEW

File: sns/cm/repair/ut/cm.c [hunk 1]
```diff
@@ -43,6 +43,7 @@
 #include "mdservice/md_fid.h"
 #include "rm/rm_service.h"                 /* m0_rms_type */
 #include "sns/cm/repair/ag.h"
+#include "sns/cm/rebalance/ag.c"
```
Comment (mehjoshi) [inline sns/cm/repair/ut/cm.c:46]: Why do we need to include this? Export the required declarations through the .h file and include that instead.

File: sns/cm/repair/ut/cm.c [hunk 2]
```diff
@@ -68,6 +69,7 @@ static struct m0_fom_simple     iter_fom;
 static struct m0_fom_timeout    iter_fom_timeout;
 static struct m0_semaphore      iter_sem;
 static const struct m0_fid      M0_SNS_CM_REPAIR_UT_PVER = M0_FID_TINIT('v', 1, 8);
+static enum m0_cm_op op;
```
Comment (mehjoshi) [inline sns/cm/repair/ut/cm.c:72]: Alignment. `op` should be right aligned.

File: sns/cm/repair/ut/cm.c (hunk 1 above)
Comment (JugalPatil) [reply sns/cm/repair/ut/cm.c]: Thanks Mehul this import was not needed. I have fixed this in the latest commit.

File: sns/cm/repair/ut/cm.c (hunk 2 above)
Comment (JugalPatil) [reply sns/cm/repair/ut/cm.c]: Done in latest commit.

File: sns/cm/repair/ut/cm.c [hunk 3]
```diff
@@ -68,6 +68,7 @@ static struct m0_fom_simple     iter_fom;
 static struct m0_fom_timeout    iter_fom_timeout;
 static struct m0_semaphore      iter_sem;
 static const struct m0_fid      M0_SNS_CM_REPAIR_UT_PVER = M0_FID_TINIT('v', 1, 8);
+static enum m0_cm_op 			op;
```
Comment (mehjoshi) [inline sns/cm/repair/ut/cm.c:71]: Spacing. op should be aligned with M0_SNS_CM_REPAIR_UT_PVER.

Comment (JugalPatil) [reply sns/cm/repair/ut/cm.c]: Done in latest commit

**Review by mehjoshi — APPROVED** [review]

**Review by nkommuri — APPROVED** [review]

**Pull request merged**